### ISSUE TITLE
Label of MultidomainType is not showing when entries are TextareaType

### ISSUE THREAD

**Issue by LuKillman** [shopsys/shopsys#1783]
### What is happening

When I add `MultidomainType` field to a form in admin and I set `entry_type` to `TextareaType`, its label is hidden behind background of the first textarea's `form-line` and remaining textareas are missing left margin. 
![image](https://user-images.githubusercontent.com/26909433/79218111-33d06400-7e50-11ea-9171-8bc5c9683a16.png)
```
    /**
     * @param \Symfony\Component\Form\FormBuilderInterface $builder
     * @param array $options
     */
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('multidomainTextareas', MultidomainType::class, [
                'entry_type' => TextareaType::class,
                'label' => t('Label'),
                'required' => false,
            ])
            ->add('save', SubmitType::class);
    }
```
### Expected result

Label should definitely be visible. It could be located above the first textarea and then textareas themselves wouldn't have left margin. Or it could be located where it is now but not covered by white background of another element and then textareas for other domains would have to have the same margin as the first.

**Comment by grossmannmartin** [comment]
Hi, thanks for the issue and the code to reproduce. This will help a lot!

**Comment by LuKillman** [comment]
I just noticed in `CategoryFormType` that the same scenario works well for `seoMetaDescriptions` field probably because of 
```
                'macro' => [
                    'name' => 'seoFormRowMacros.multidomainRow',
                    'recommended_length' => 155,
                ]
```
but for some reason I haven't noticed it earlier. 
But I wanted to add that for `YesNoType` inside of `MultidomainType` formatting also doesn't work very well. 
![image](https://user-images.githubusercontent.com/26909433/79467163-c95d2680-7ffd-11ea-817a-03adfd49ecc9.png)

**Comment by pk16011990** [comment]
Selectbox in MultidomainType causes very similar problem with design...

**Comment by grossmannmartin** [comment]
resolved by #2348
